cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -77,7 +83,8 @@ def add_user_info(self, user_info):
                 'GradSemester': user_info.get('GradSemester', ' '),
                 'GradYear': user_info.get('GradYear', ' '),
                 'Major': ', '.join(sorted(user_info.get('Major', []))),
-                'Minor': ', '.join(sorted(user_info.get('Minor', [])))
+                'Minor': ', '.join(sorted(user_info.get('Minor', []))),
+                'ttl_expiration':ttl_expiration,
```

Comment:
Nitpick but this should adhere to the same style as the rest of the table. TtlExpriation

Reply:
I think I can change this one to keep the same formatting.

Reply:
In your latest commits it looks like you didn’t take this suggestion. Is that intentional?

Reply:
For using the "last_updated" format instead of something like "LastUpdated", I think it is better to keep the TTL attribute names uniform across all DynamoDB tables. While it could be helpful for this one table to have uniform formatting, I think keeping every ttl attribute name the same (as "last_updated") would probably be simpler if future teams need to interact with the items.